overlord/servicestate/quota_handlers.go

```diff
@@ -222,6 +220,80 @@ func addRestartServicesTasks(st *state.State, queueTask func(task *state.Task),
 	}
 }
 
+func (m *ServiceManager) doAddSnapToQuota(t *state.Task, _ *tomb.Tomb) error {
+	st := t.State()
+	st.Lock()
+	defer st.Unlock()
+
+	perfTimings := state.TimingsForTask(t)
+	defer perfTimings.Save(st)
+
+	var quotaGroupName string
+	var snapNames []string
+	if err := t.Get("quota-on-install-snapnames", &snapNames); err != nil {
```

Comment:
we never put the name of the task in the parameter name, also I'm not sure why this should support multiple snaps? we have separate lanes per snaps in our install operation, so the normal thing would be to have one task per snap.

This then would be a single "snap-name"